doc/guide/guide-bloch.rst

```diff
@@ -443,4 +443,4 @@ The code to directly generate an mp4 movie of the Qubit decay is as follows ::
                                  init_func=init, blit=False, repeat=False)
    ani.save('bloch_sphere.mp4', fps=20)
 
-The resulting movie may be viewed here: `bloch_decay.mp4 <https://raw.githubusercontent.com/qutip/qutip/master/doc/figures/bloch_decay.mp4>`_
+he resulting movie may be viewed here: `bloch_decay.mp4 <https://raw.githubusercontent.com/qutip/qutip/master/doc/figures/bloch_decay.mp4>`_
```

Comment:
```suggestion
The resulting movie may be viewed here: `bloch_decay.mp4 <https://raw.githubusercontent.com/qutip/qutip/master/doc/figures/bloch_decay.mp4>`_
```
Not sure why the T got deleted here, but I guess it should be put back. :)

Reply:
Honest mistake, haha.